packages/dm-core-plugins/src/attribute-selector/README.md

```diff
@@ -5,4 +5,13 @@ in a new tab.
 
 If you want to specify a config object in some UiRecipe-entity, the Blueprint is included in the npm package.
 
+
+The attribute-selector UI plugin can accept a config of type `AttributeSelectorPluginConfig` with the following attributes:
+
+* `childTabsOnRender` (optional): ??
```

Comment:
Could not figure out what childTabsOnRender is used for...

Reply:
I think that if `childTabsOnRender == false`, then the tab/sidebar will only show the home button and not the children. I suppose we don't need it anymore, now that we got the `visibleAttributes`? We could make `visibleAttributes` so that if `visibleAttributes == undefined` we show all attributes, but if `visibleAttributes == []`we show none of them.

Reply:
I think "null | undefined| []" should be treated the same. Showing all attributes.
Since the default value for "visibleAttributes" is "[]", that should result in showing all.
Goal being that "stuff just works" when there is no special config and default values.

Reply:
Can't we just change the default value to null, then?

Reply:
I dont think we should remove "childTabsOnRender". Could still be useful for entities with a ton of attributes.
One problem is that default values for lists will always be "[]", as instantiated from DMSS.

Reply:
```suggestion
* `childTabsOnRender` (optional): If false the tab/sidebar will only show the home button and not the attributes. Defaults to true.
```